### PR TITLE
feat: add descriptions and enhance carousel

### PR DESCRIPTION
**Pull request by chellysA**

### CODE REVIEW

File: src/constants/swordvoices.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ const swordvoices = [
         photo: "/swordvoices/jainer.jpeg",
         name: "Jainer Munoz",
         description:
-          "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation",
+          "CEO, Consultor IT con 10+ años de experiencia, 5 años de desarrollo web moderno y gestión de equipos. Gran trabajador en entornos de equipo multiculturales.",
```
Comment (Yireh2016) [inline src/constants/swordvoices.tsx]: CEO, Consultor IT y negocios con 10+ años de experiencia, 8 años de desarrollo web moderno y gestión de equipos en entornos multiculturales.

File: src/constants/swordvoices.tsx [hunk 2]
```diff
@@ -33,8 +33,8 @@ const swordvoices = [
       photo: "/swordvoices/chellys.jpg",
       name: "Chellys Castillo",
       description:
-        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation",
-      rol: "Frontend Dev Jr",
+        "Mentora, eficiente en el desarrollo de interfaces de usuarios, minusiosa y creativa en el diseño pixel to pixel y responsivo de apps.",
```
Comment (Yireh2016) [inline src/constants/swordvoices.tsx]: "pixel to pixel"

**Review by Yireh2016 — CHANGES_REQUESTED** [review]

**Comment by Yireh2016** [comment]
should we do these reviews in english? That would be great for practicing too.

**Comment by Yireh2016** [comment]
Check merge conflicts ?

File: src/app/page.tsx [hunk 3]
```diff
@@ -240,7 +243,7 @@ export default function Home() {
       </motion.div>
       <div className="h-[58px] bg-[black] w-full relative top-3"></div>
       {/* Fifth Page */}
-      <div id="our_paths" className="h-auto xl:min-h-screen bg-white w-full">
+      <div id="our_paths" className="h-auto xl:min-h-screen bg-white w-full ">
```
Comment (Yireh2016) [inline src/app/page.tsx]: espacio de mas al final...pudieran usar un prettier/eslint para que automaticamente borre estos espacios al salvar los archivos.

File: src/constants/swordvoices.tsx (hunk 1 above)
Comment (chellysA) [reply src/constants/swordvoices.tsx]: actualiza tu linkedlin jaja

**Comment by chellysA** [comment]
> should we do these reviews in english? That would be great for practicing too.

yes, i should do the entire page in english, it is tomorrow task.

**Comment by Yireh2016** [comment]
> > should we do these reviews in english? That would be great for practicing too.
> 
> yes, i should do the entire page in english, it is tomorrow task.

Me refiero a los reviews

**Review by Yireh2016 — APPROVED** [review]

**Review by ilemarandrade — APPROVED** [review]

**Pull request merged**